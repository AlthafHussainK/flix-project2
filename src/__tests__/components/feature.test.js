import React from 'react'
import { render } from '@testing-library/react'
import { Feature } from '../../components'

describe('<Feature />', () => {
  it('renders the <Feature /> with populated data', () => {
    const { container, getByText } = render(
      <Feature>
        <Feature.Title>Unlimited films, TV programmes and more.</Feature.Title>
        <Feature.SubTitle>Watch anywhere. cancel at any time.</Feature.SubTitle>
      </Feature>
    )

    expect(getByText('Unlimited films, TV programmes and more.')).toBeTruthy()
    expect(getByText('Watch anywhere. cancel at any time.')).toBeTruthy()
    expect(container.firstChild).toMatchSnapshot()
    
  })
})
