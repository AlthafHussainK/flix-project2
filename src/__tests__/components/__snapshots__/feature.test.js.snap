// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Feature /> renders the <Feature /> with populated data 1`] = `
<div
  class="feature__Container-sc-6swrp6-0 isjLws"
>
  <h1
    class="feature__Title-sc-6swrp6-1 biQKvD"
  >
    Unlimited films, TV programmes and more.
  </h1>
  <h2
    class="feature__SubTitle-sc-6swrp6-2 ezZeoY"
  >
    Watch anywhere. cancel at any time.
  </h2>
</div>
`;
